### ISSUE TITLE
SSH: fwknop

### ISSUE THREAD

**Issue by lkraav** [imthenachoman/How-To-Secure-A-Linux-Server#32]
Once I learned the setup, I'm never going back.

http://cipherdyne.org/fwknop/

**Comment by imthenachoman** [comment]
Looks interesting. I will add to the to do list but it might take a while before I have time to figure out how to set it up. Thanks!

**Comment by lkraav** [comment]
I can provide some kickoff pointers, when I find an afternoon with some brainspace to recall.

**Comment by pahakalle** [comment]
@lkraav Does fwknop need to run on its own server/vm seperate to the ones its protecting or did i understand it wrong?

**Comment by lkraav** [comment]
No, it doesn't need to, but in my case I am running it on my OpenWRT router, rather than the protected intranet server.
